test/reactionsystem_structure/reactionsystem.jl

```diff
@@ -1,15 +1,24 @@
 ### Fetch Packages and Set Global Variables ###
 
-# Fetch pakcages.
-using Catalyst, LinearAlgebra, JumpProcesses, Test, OrdinaryDiffEq, StochasticDiffEq
-t = default_t()
+# Fetch packages.
```

Comment:
There is a couple of really long tests here that I didn't want to touch. They pass, but at some point it would be good if we could look at them together to decide how they can be remade.